hack/integration.sh

```diff
@@ -22,10 +22,10 @@ stats() {
   ${REDSKYCTL_BIN} generate experiment -f hack/app.yaml
   echo "::endgroup::"
   echo "::group::Describe Application Resources"
-  kubectl describe trials,jobs,pods -l redskyops.dev/application=ci
+  kubectl describe trials,jobs,pods -l stormforge.io/application=ci
```

Comment:
I rebased `part-deux` onto `master` and picked up this mismatched label that needed to be corrected.